wordpress-mobile/WordPress-iOS
# [Insights Management] Disable already displayed Insights

Ref #12246 

To test:
- Go to Stats > Insights.
- Select 'Add new stats card'.
- Verify the stats already displayed are greyed out and disabled (i.e. selecting them does nothing).
- Select an available stat.
- Verify it appears in the Insights table.
- Return to 'Add new stats card'.
- Verify the newly selected stat is now greyed out.

![selected_stats](https://user-images.githubusercontent.com/1816888/66246609-89be9500-e6d2-11e9-926d-f9253cc4f209.gif)

Update release notes:
